wallet-new/server/Cardano/Wallet/API/V1/LegacyHandlers/Wallets.hs

```diff
@@ -23,18 +23,13 @@ handlers :: ( HasConfigurations
             , HasCompileInfo
             )
          => ServerT Wallets.API MonadV1
-handlers =
-        (    newWallet
-        :<|> listWallets
-        :<|> (\walletId ->
-                     updatePassword walletId
-                :<|> deleteWallet walletId
-                :<|> getWallet walletId
-                :<|> updateWallet walletId
-             )
-        )
-        -- NOTE Separation between tags ("Wallets" vs "Accounts"), see Cardano.Wallet.API.V1.Wallets
-        :<|> (\walletId -> Accounts.handlers walletId)
+handlers = (newWallet
+    :<|> listWallets
+    :<|> updatePassword
+    :<|> deleteWallet
+    :<|> getWallet
+    :<|> updateWallet
+    ) :<|> Accounts.handlers
```

Comment:
Why not the simpler version without the parens as in https://github.com/input-output-hk/cardano-sl/commit/4216cdcba2612d95d98a90b014353d50040dcaf1#diff-a6de663b7f7ec63d491ede6773183ddfR32 ?

Reply:
I want to say that some of the `Tags` or `Summary` stuff get weird with that. I'll try it out and see if it works.

Reply:
Probably that's the reason -- I think @KtorZ had a similar problem whilst working on CBR-102.